src/openrct2/paint/tile_element/Paint.Entrance.cpp

```diff
@@ -270,20 +271,20 @@ static void park_entrance_paint(paint_session* session, uint8_t direction, int32
                 break;
 
             {
-                set_format_arg(0, uint32_t, 0);
-                set_format_arg(4, uint32_t, 0);
+                Formatter::Common().Add<uint32_t>(0).Add<uint32_t>(0);
```

Comment:
Should this get the same comment as #11388 ?

Reply:
I wouldn't be able to tell. I don't know if the arguments are cleared for the same reason.